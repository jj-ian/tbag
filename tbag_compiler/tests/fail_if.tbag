/* Authors: Maria */
true {
	testif();
	endgame;
}


func int testif() {
	if ("hello") {
		print("true ");
	}
	else {}
	print("hi");
	return 0;
}

