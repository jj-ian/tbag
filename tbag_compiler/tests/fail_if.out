Fatal error: exception Failure("Expression in if statement conditional must be
    of type Boolean")
